src/endpoint.cpp

```diff
@@ -1179,18 +1179,18 @@ bool UdpEndpoint::validate_config(const UdpEndpointConfig &config)
         return false;
     }
 
-    if (config.mode == UdpEndpointConfig::Mode::Server && config.port == ULONG_MAX) {
-        log_error("UdpEndpoint %s: Port is required in Server mode", config.name.c_str());
-        return false;
-    }
-
```

Comment:
why is this in this commit?

Reply:
Looks like a wrong squash.... should be in the commit that cleanup the defaults?

Reply:
It was something, that came up during testing that method, so I added the fix to the unit test commit. I can move it, if you like.